corehq/apps/accounting/migrations/0049_app_user_profiles.py

```diff
@@ -0,0 +1,26 @@
+from django.db import migrations
+from django.core.management import call_command
+from corehq.util.django_migrations import skip_on_fresh_install
+
+from corehq.privileges import APP_USER_PROFILES
+
+
+@skip_on_fresh_install
+def _grandfather_basic_privs(apps, schema_editor):
+    call_command('cchq_prbac_bootstrap')
+    call_command(
+        'cchq_prbac_grandfather_privs',
+        APP_USER_PROFILES,
+        skip_edition='Paused,Community,Standard,Pro',
```

Comment:
Shouldn't this also skip advanced plans?

Reply:
Yes!-- definitely an oversight